ansible/vars/swiss/swiss_public.yml

```diff
@@ -21,6 +21,10 @@ active_sites:
   riakcs: False
   couchdb2: False
 
+couchdb2:
```

Comment:
Swiss doesn't have this yet right? also would need the cluster settings I think

Reply:
good catch. I'm not sure whether it makes a difference, but yeah I shouldn't have included this commit in this PR. Reverting here: https://github.com/dimagi/commcarehq-ansible/pull/968